CHANGELOG.md

```diff
@@ -41,6 +41,11 @@
 ### Fixed
 * Writing a snapshot to file via Group::write() could produce a file with some parts not
   reachable from top array (a memory leak). ([#2911](https://github.com/realm/realm-sync/issues/2911))
+* SharedGroup::open() could fail with indications of a file corruption due to a race with the page reclaimer.
```

Comment:
@finnschiermer Isn't this added in the wrong place?
Also suggest this end-user description:
"When opening an encrypted file via `SharedGroup::open()`, it could wrongly fail and indicate a file corruption although the file was ok. ([#3267](https://github.com/realm/realm-core/issues/3267), since core v5.12.2)"